docs/source/features/run_workflow_remotely.md

```diff
@@ -35,3 +31,18 @@ python -m olive run --config <config_file.json>
 ```
 
 Olive will submit this workflow to the Azure ML system compute. It is safe to close your terminal session or turn off your local machine. The workflow will continue running on the Azure ML workspace compute.
+
+## Workflow outputs
+
+The artifacts of the workflow, including cache and outputs, will be automatically exported to Azure Machine Learning datastore. The default datastore is `workspaceblobstore`. If you want to export to your own datastore, please add `datastore: <your-datastore-name>` in AzureML system config:
+
+```json
+"aml_system": {
+    "type": "AzureML",
+    "config": {
+        "datastore": "<datastore-name>"
+    }
+}
+```
+
+The cache and outputs will be exported to `<datastore>/workflow_artifacts/<workflow_id>/cache` and `<datastore>/workflow_artifacts/<workflow_id>/output`.
```

Comment:
can you update aml system to also log the resolved path. The user has no information about the default values of `datastore` and `workflow_id`

Reply:
Updated